includes/Data/ContentBlocksResolver.php

```diff
@@ -99,12 +83,12 @@ static function ( $parsed_block ) {
 			},
 			$parsed_blocks
 		);
-
+		// Resolve reusable blocks - replaces "core/block" with the corresponding block(s) from the reusable ref ID
+		TraverseHelpers::traverse_blocks( $parsed_blocks, [ 'WPGraphQL\ContentBlocks\Utilities\TraverseHelpers', 'replace_reusable_blocks' ], 0, PHP_INT_MAX );
```

Comment:
Do you mind referencing classes using `\ClassName::class` instead of unvalidated strings? It really helps DX.